docs/pages/setup/guides/docker.mdx

```diff
@@ -9,13 +9,17 @@ This section will cover:
 - Getting started with a local Teleport using Docker.
 - Using Teleport with Teleport's native client, `tsh`.
 
+<Notice scope={["oss", "enterprise"]} type="note" title="Enterprise using Docker" scopeOnly={true}>
```

Comment:
Should the scope only include `oss` since this is an open-source guide?

Reply:
I'd say no since you want someone to see this notice if they're in enterprise scope.  otherwise they wouldn't know not to use this guide in a enterprise context.

Reply:
Should we only include this Notice for Enterprise users? For OSS users, it might be a bit noisy.